fastlane_core/spec/device_manager_spec.rb

```diff
@@ -0,0 +1,264 @@
+require 'open3'
+
+describe FastlaneCore do
+  describe FastlaneCore::DeviceManager do
+    before do
+      @valid_simulators = File.read('./spec/fixtures/DeviceManagerSimctlOutput')
+      FastlaneCore::Simulator.clear_cache
+    end
```

Comment:
suggestion: change this to a `before(:all)` block and move all fixture file loading in. This way we can just read the canned responses once, instead of before each individual test.